go.mod

```diff
@@ -132,8 +132,26 @@ replace (
 	k8s.io/apimachinery => k8s.io/apimachinery v0.22.15
 	k8s.io/client-go => k8s.io/client-go v0.22.15
 
+	// // Pinning to 0.22.15 because of these errors
+	// # k8s.io/component-base/config
+	// ../../go/pkg/mod/k8s.io/component-base@v0.23.5/config/types.go:130:26: undefined: resource.QuantityValue
+	// make: *** [Makefile:51: build-foghorn] Error 2
+	k8s.io/component-base => k8s.io/component-base v0.22.15
+
 	// controller-runtime has k8s 1.22 in go mod
-	sigs.k8s.io/controller-runtime => sigs.k8s.io/controller-runtime v0.10.3
+	// Pinning to 0.11.2 because of these errors
+	// ../../go/pkg/mod/sigs.k8s.io/controller-runtime@v0.10.3/pkg/log/log.go:67:16: cannot use NullLogger{} (value of type NullLogger) as type logr.Logger in argument to Log.Fulfill
+	// ../../go/pkg/mod/sigs.k8s.io/controller-runtime@v0.10.3/pkg/log/log.go:82:31: cannot use NullLogger{} (value of type NullLogger) as type logr.Logger in argument to NewDelegatingLogger
+	// ../../go/pkg/mod/sigs.k8s.io/controller-runtime@v0.10.3/pkg/log/log.go:86:24: cannot use Log (variable of type *DelegatingLogger) as type logr.Logger in variable declaration
+	// ../../go/pkg/mod/sigs.k8s.io/controller-runtime@v0.10.3/pkg/log/log.go:88:16: assignment mismatch: 1 variable but logr.FromContext returns 2 values
+	// ../../go/pkg/mod/sigs.k8s.io/controller-runtime@v0.10.3/pkg/log/null.go:30:21: cannot use NullLogger{} (value of type NullLogger) as type logr.Logger in variable declaration
+	// ../../go/pkg/mod/sigs.k8s.io/controller-runtime@v0.10.3/pkg/log/null.go:49:9: cannot use log (variable of type NullLogger) as type logr.Logger in return statement
+	// ../../go/pkg/mod/sigs.k8s.io/controller-runtime@v0.10.3/pkg/log/deleg.go:79:17: undefined: logr.WithCallDepth
+	// ../../go/pkg/mod/sigs.k8s.io/controller-runtime@v0.10.3/pkg/log/deleg.go:163:9: cannot use res (variable of type *DelegatingLogger) as type logr.Logger in return statement
+	// ../../go/pkg/mod/sigs.k8s.io/controller-runtime@v0.10.3/pkg/log/deleg.go:179:9: cannot use res (variable of type *DelegatingLogger) as type logr.Logger in return statement
+	// ../../go/pkg/mod/sigs.k8s.io/controller-runtime@v0.10.3/pkg/log/deleg.go:195:9: cannot use res (variable of type *DelegatingLogger) as type logr.Logger in return statement
+	// ../../go/pkg/mod/sigs.k8s.io/controller-runtime@v0.10.3/pkg/log/null.go:49:9: too many errors
+	sigs.k8s.io/controller-runtime => sigs.k8s.io/controller-runtime v0.11.2
 )
 
-go 1.18
+go 1.19
```

Comment:
This resulted in no changes to the mod/sum files, all the changes are from tekton upgrade to 0.33.4